test/integration/targets/copy/tasks/main.yml

```diff
@@ -169,8 +169,9 @@
 # Test copy with mode=preserve
 - name: Set file perms to an odd value
   file:
-    path: '{{ output_file }}'
+    path: '{{ role_path }}/files/foo.txt'
```

Comment:
I do not like doing this because then it's changing the mode of a file that is checked into the git repo.